extensions/resource-deployment/package.json

```diff
@@ -391,6 +391,338 @@
             "when": "version=sql2019"
           }
         ]
+      },
+      {
+        "name": "azure-sql-vm",
+        "displayName": "%azure-sqlvm-display-name%",
+        "description": "%azure-sqlvm-description%",
+        "platforms": "*",
+        "icon": {
+          "light": "./images/azure-sql-vm.svg",
+          "dark": "./images/azure-sql-vm.svg"
+        },
+        "providers": [
+          {
+            "notebookWizard": {
+              "notebook": "./notebooks/azurevm/create-sqlvm.ipynb",
+              "runNotebook": false,
+              "codeCellInsertionPosition": 4,
+              "taskName": "%azure-sqlvm-deploy-dialog-title%",
+              "actionText": "%azure-sqlvm-deploy-dialog-action-text%",
+              "title": "%azure-sqlvm-deploy-dialog-title%",
+              "name": "azure.sqlvm.new.wizard",
+              "generateSummaryPage": true,
+              "pages": [
+                {
+                  "title": "%azure-sqlvm-azure-account-page-label%",
+                  "sections":
+                  [
+                    {
+                      "title": "",
+                      "collapsible": false,
+                      "fields": [
+                        {
+                          "subscriptionVariableName": "AZDATA_NB_VAR_AZURE_SQLVM_NB_VAR_SUBSCRIPTION",
+                          "resourceGroupVariableName": "AZDATA_NB_VAR_AZURE_SQLVM_NB_VAR_RESOURCE_GROUP_NAME",
+                          "type": "azure_account",
+                          "required": true
+                        },
+                        {
+                          "type": "azure_locations",
+                          "label": "%azure-sqlvm-azure-location-label%",
+                          "defaultValue": "eastus",
+                          "required": true,
+                          "locationVariableName": "AZDATA_NB_VAR_AZURE_SQLVM_LOCATION",
+                          "displayLocationVariableName": "AZURE_SQLVM_DISPLAY_LOCATION",
```

Comment:
Most of the vars start with AZDATA_NB_VAR_AZURE_SQLVM, but not this one. Intended?